### ISSUE TITLE
Questions about maxdepth

### ISSUE THREAD

**Issue by bradasteiner** [ialhashim/DenseDepth#25]
Hi, I have a few questions, mostly regarding maxdepth:

1. The maxdepth value seems to be set to 1000 throughout the code, however I noticed that the depth values in the NYU dataset range all the way up to 10000. Should the maxdepth value be changed to 10000 when training NYU?

2. When making predictions, the predicted depth values can be higher than any values seen in the training set. Do you know why this is so? Do I need to normalize it to the range of the training data as a post processing step?

3. Lastly, when training NYU I am noticing that the validation loss does not decrease at all, but the training loss does steadily. Any ideas why this might be happening?

Thanks!

**Comment by ialhashim** [comment]
1. The depth values are in the range of up to 10 meters max. If its either 1000 or 10000 it won't matter as long as you remain consistent about what the maximum and minimum depths are. If you use a different dataset just make sure the ranges are correct and any computation in the data processing, loss, or visualization is correct.

2. For regression its expected that values go bellow or over the expected range. However, you do need to clip the values to the ranges of the training data. Any depth value greater than 10 M is not to be taken seriously (its most likely not accurate).

3. Training loss typically always go down since that is what the optimizer is solving for. Validation loss typically decreases very slowly and in case of over-fitting it might stay the same or go higher. If you train and use the '-full' flag the testing data will be evaluated. You can use that as your 'validation' although its not a real indicator of the model's generalization behavior or its performance in real world cases. One thing to double check is the size of the validation set used.

**Comment by boni-hu** [comment]
Hi ,

I have a question about the first question, the depth values of the nyu dataset are between 18 to 255, and the value is between 0.1 to 10 meters in the real world. Can i regards it is just a different Coordinate System. So if i train my own data which have the different depth range of nyu dataset in the real world and i also transformed it to 0~255. Can i still use the max_depth and the min_depth in this code. Because of the great performance with this scale, and then i just transformed the test results from(10~1000) to my own scale. 

Thanks!
